script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -523,42 +562,45 @@ She's serious.
 [sha:0d41371da5b23dc3416a513bb039210f8b84573f]{
 -- Page 163, Offset 33653.
 -- この人は、本気で、俺を殺すつもりだ。
-She's seriously intending on killing me.
+She's seriously intent on killing me.
 }
 [sha:de8a33e305a0cb46c6d0c87c1d9508adbf4943c3]{
 -- Page 164, Offset 33654.
 -- ―――ギッ。
-―――Gh.
+―――%{i}Gh%{/i}.
 }
 [sha:1769fe7c6807035a583ec85663571f091d0f9a3e]{
 -- Page 164, Offset 33655.
 -- 殺される運命を感じとって、神経が軋みをあげる。
-Realizing that I'm destined to be killed, my nerves creak.
+My nerves start squealing,
+ sensing that I'm about to be killed.
```

Comment:
Not sure that straining on its own works here - they're _under_ strain, nerves themselves don't move. Squealing is a fairly associated word for that, since when you put almost breaking strain on metals / plastics they'll squeal before they snap, which I assume is the origin of the phrase "squealing under the pressure"

Reply:
I don't really understand the objection?

Reply:
Basically that "my nerves begin to strain" makes it sounds like the nerves are motile, when I think the actual meaning is more like they are creaking _under_ the strain of knowing death is imminent? I'd use 'stretched to breaking point' here but he went and said pretty much that in the line that comes after it. 'My nerves are screaming at me' is the other EN-idiomatic way I'd probably phrase this if that works better?

Reply:
Squealing is def out, but "nerves straining" doesn't read to me as you said it is, I guess? If anything the suggestion does. Regardless, strain and nerve are very close collocations buddies, so I think it works.

If that's unacceptable, sure, screaming is much better.